src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -185,11 +185,6 @@ public void updateFilteredIncomeList(Predicate<Entry> predicate) {
         filteredIncome.setPredicate(predicate);
     }
 
-    @Override
-    public void setMonthForChart(YearMonth month) {
```

Comment:
Hi @yuanxi1, could I check if we need this method?

Reply:
Ok! I will add it back

Reply:
sorry for the premature merge! wanted to merge in to find out which test cases need to be worked on exactly.

@EmilyOng Rh is fixing up some CI issues, can sync your fork after his PR has been merged in before opening this one!! thanks 😄